Dalamud/Game/Addon/Lifecycle/AddonLifecycle.cs

```diff
@@ -169,13 +174,19 @@ private void OnAddonSetup(AtkUnitBase* addon, uint valueCount, AtkValue* values)
         try
         {
             // Hook the addon's ReceiveEvent function here, but only enable the hook if we have an active listener.
+            // Disallows hooking the core internal event handler.
             var addonName = MemoryHelper.ReadStringNullTerminated((nint)addon->Name);
-            var receiveEventHook = Hook<AddonReceiveEventDelegate>.FromAddress((nint)addon->VTable->ReceiveEvent, this.OnReceiveEvent);
-            this.receiveEventHooks.TryAdd(addonName, receiveEventHook);
-
-            if (this.eventListeners.Any(listener => (listener.EventType is AddonEvent.PostReceiveEvent or AddonEvent.PreReceiveEvent) && listener.AddonName == addonName))
+            var receiveEventAddress = (nint)addon->VTable->ReceiveEvent;
+            Log.Debug($"{receiveEventAddress:X} - {this.disallowedReceiveEventAddress:X}");
```

Comment:
Remove this log entry, or set to verbose with more descriptive text.

Reply:
Oops, nice catch.